src/rows.js

```diff
@@ -68,6 +69,35 @@ export class Rows {
     return { value: row, done: false };
   }
 
+  /**
+   * Rows.columns
+   *
+   * Call this if you need column names from the result of a select query.
+   *
+   * This method returns an array of objects, where each object has the following properties.
+   *
+   * | Property     | Value                                      |
+   * |--------------|--------------------------------------------|
+   * | `name`       | the result of `sqlite3_column_name`        |
+   * | `originName` | the result of `sqlite3_column_origin_name` |
+   * | `tableName`  | the result of `sqlite3_column_table_name`  |
+   */
+  columns() {
+    if (this._done) {
+      throw new SqliteError("Unable to retrieve column names as transaction is finalised.")
```

Comment:
Nit: can we spell this `finalized` to reflect the SQLite3 spelling in `sqlite3_finalize`? (s is not technically wrong, but I'd like to be consistent with the SQLite3 docs)